recipe/meta.yaml

```diff
@@ -14,13 +14,11 @@ source:
     sha256: a9598abd5c697e4a49a512b285c5747413d1f45155a12ecb1f5ee389c0459afa
     folder: ucx
     patches:
-      - patches/PR_4137.patch
-      - patches/PR_4366.patch
       - patches/PR_4383.patch
       - patches/PR_4384.patch
-      - patches/PR_4431.patch
+      - patches/PR_4515.patch
```

Comment:
https://github.com/openucx/ucx/pull/4515 is already merged, I believe we can remove it from here?

Reply:
Was there a new RC release made that includes this patch? If not, I think we still need it for that reason.

Reply:
https://github.com/conda-forge/ucx-split-feedstock/pull/48/files#r352912898

Reply:
Happy to merge or wait. No strong preference.

Do we know what their ETA is for RC2? Is there an issue I should be following?

Reply:
It was mentioned before that that PR was the only one being waited for RC2, I commented there https://github.com/openucx/ucx/pull/4515#issuecomment-560920527. If you're ok with it, I expect they won't take long to tag so I would suggest waiting until tomorrow and then merging with or without RC2. I would be slightly more inclined to wait, as we then get as close to 1.7.0 as it gets.

Reply:
Yeah I'm ok waiting. Maybe we still need another patch ( https://github.com/openucx/ucx/pull/4530 ) actually.

Reply:
Also found another missing patch while making this. Backporting in PR ( https://github.com/openucx/ucx/pull/4533 ).

Reply:
Is this last one relevant for us? Just asking because I don't think I've experienced an issue with that so far.

Reply:
It seems like without it we would have compilation issues. Am surprised it hasn't actually.